assignments/IO-assignment/solution/Step3.rs

```diff
@@ -0,0 +1,17 @@
+use std::io::{Read, BufReader, BufRead, Lines};
+use std::fs::File;
+
+fn main() {
+
+    let f = File::open("src/lib/content.txt");
+
+    let file = match f {
+        Ok(file) => file,
+        Err(e) => panic!("Problem opening the file: {:?}", e),
+    };
+
+    let mut buf_reader = BufReader::new(file);
+    let mut content_string = String::new();
+    buf_reader.read_to_string(&mut content_string).unwrap();
```

Comment:
`fn main() -> std::io::Result<()>` would allow us to use `?` instead of unwrap here.